### PR TITLE
Add linting to unit test files

### PR DESCRIPTION
**Pull request by jasonnutter**
In addition to improving code quality, this will help us catch when someone accidently commits an exclusive test.

### CODE REVIEW

File: lib/msal-browser/test/app/PublicClientApplication.spec.ts [hunk 1]
```diff
@@ -25,12 +25,11 @@ import { SilentRequest } from "../../src/request/SilentRequest";
 import { BrowserCacheManager } from "../../src/cache/BrowserCacheManager";
 import { RedirectRequest } from "../../src/request/RedirectRequest";
 import pkg from "../../package.json";
-import { ClientApplication } from "../../src/app/ClientApplication";
 
 chai.use(chaiAsPromised);
 const expect = chai.expect;
 
-describe("PublicClientApplication.ts Class Unit Tests", () => {
+describe.skip("PublicClientApplication.ts Class Unit Tests", () => {
```
Comment (jasonnutter) [inline lib/msal-browser/test/app/PublicClientApplication.spec.ts:32]:
```suggestion
describe("PublicClientApplication.ts Class Unit Tests", () => {
```

File: .eslintrc.json [hunk 2]
```diff
@@ -42,6 +44,10 @@
         "eol-last": 2,
         "eqeqeq": 2,
         "header/header": [2, "block", [ "", " * Copyright (c) Microsoft Corporation. All rights reserved.", " * Licensed under the MIT License.", " " ], 2],
+        "jasmine/no-disabled-tests": 1,
+        "jasmine/no-focused-tests": 2,
+        "jest/no-disabled-tests": 1,
+        "jest/no-focused-tests": 2,
```
Comment (tnorling) [inline .eslintrc.json:50]: Do we need to add mocha as well?

**Review by tnorling — APPROVED** [review]
Love this, big fan!

**Review by jo-arroyo — APPROVED** [review]
Looks great!

File: .eslintrc.json (hunk 2 above)
Comment (pkanher617) [reply .eslintrc.json:50]: ^^

Comment (pkanher617) [reply .eslintrc.json:50]: @jasonnutter Can we add mocha to this? I will work on migrating common and browser to jest at some point this quarter.

Comment (jasonnutter) [reply .eslintrc.json:50]: Don't need both, as they both use the same grammar for skipped and disabled tests.

**Review by pkanher617 — APPROVED** [review]

**Pull request closed without merging**